components/pipedrive/actions/add-organization/add-organization.mjs

```diff
@@ -1,61 +1,57 @@
-// legacy_hash_id: a_WYiEeg
-import { axios } from "@pipedream/platform";
+import pipedriveApp from "../../pipedrive.app.mjs";
 
 export default {
   key: "pipedrive-add-organization",
   name: "Add Organization",
-  description: "Adds a new organization.",
-  version: "0.1.1",
+  description: "Adds a new organization. See the Pipedrive API docs for Organizations [here](https://developers.pipedrive.com/docs/api/v1/#!/Organizations)",
+  version: "0.1.2",
   type: "action",
   props: {
-    pipedrive: {
-      type: "app",
-      app: "pipedrive",
-    },
-    companydomain: {
-      type: "string",
-      description: "Your company name as registered in Pipedrive, which becomes part of Pipedrive API base url.",
-    },
+    pipedriveApp,
     name: {
       type: "string",
+      label: "Name",
       description: "Organization name",
     },
-    owner_id: {
-      type: "integer",
-      description: "ID of the user who will be marked as the owner of this organization. When omitted, the authorized user ID will be used.",
-      optional: true,
+    ownerId: {
+      propDefinition: [
+        pipedriveApp,
+        "ownerId",
+      ],
```

Comment:
`ownerId` should use async options. Could this use the same options as `userId`?